client/app/src/main/res/values/strings.xml

```diff
@@ -27,4 +27,16 @@
     <string name="filter_requests_checkbox">Requested</string>
     <string name="filter_accepted_checkbox">Accepted</string>
     <string name="filter_borrowed_checkbox">Borrowed</string>
+
+    <!--  Book Activity  -->
+    <string name="book_title_placeholder">Title</string>
+    <string name="book_detail">Detail: </string>
+    <string name="book_author">Author: </string>
+    <string name="book_isbn">ISBN: </string>
+    <string name="book_status">Status: </string>
+
+    <!--  My Book Activity  -->
```

Comment:
could rename it to `My Book Details Activity` so its a bit more different from the other one